dvc/version.py

```diff
@@ -11,7 +11,7 @@
 
 
 def _generate_version(base_version):
-    """Generate a version with information about the git repository"""
+    """Generate a version with information about the Git repository."""
```

Comment:
To be honest, not a big fan of such changes in the code. It only pollutes the history and doesn't provide any value.

Reply:
I'm used to making copy edits on docs all the time so this is kind of automatic for me but whatever policies you tell me for the core repo I'll abide to the best of my ability. Personally I don't understand the problem of history pollution but I guess it complicates your release management? Anyway, I'll follow your lead on these things of course.